### DISCUSSION TITLE
trouble getting proxy on old browser

### DISCUSSION THREAD

**Discussion by nyctophase** [atauenis/webone#153] [Q&A]
hi I am having trouble getting proxy on older browser the proxy is running but I cannot connect to the proxy on a windows 7 laptop.

**Comment by tomrow** [comment]
What browser are you using?

For Internet Explorer and Chrome-based browsers:
Open Internet Options in the Control Panel (or type `inetcpl.cpl` into the command prompt) 
Click the Connections tab, then click the LAN Settings button at the bottom of the page
Check "Use a proxy for your LAN", then type the IP address and port of your proxy into the fields below.
